server/src/entity/InventoryItem.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToMany,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { MenuItem } from './MenuItem';
+
+export enum itemCategory {
+  UNKNOWN = 'Unknown',
+  VEGETABLE = 'Vegetable',
+  MEAT = 'Meat',
+  DAILY = 'Daily',
+  FRUIT = 'Fruit',
+  OTHERs = 'Others',
+}
+
+@Entity()
+export class InventoryItem extends BaseEntity {
```

Comment:
Couldn't we just call these ingredients?

Reply:
No There can be other stuff such as fork, pan, pot

Reply:
then why would be including fork, pan, pot, etc. in a menu item?

Reply:
not necessarily including them in menu item, but maybe needed when you just want to see all the stuff you have in kitchen. Inventory is a general term